src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -1229,6 +1229,17 @@ components:
           type: string
           description: GSS code of the ERO responsible for the application
           example: E12345678
+        sourceReference:
+          type: string
+          pattern: '^[a-fA-F\d]{24}$'
+          description: |
+            A 24 character hex string. 
+            The unique identifier of an Anonymous elector document application.
+          example: 1f0f76a9a66f438b9bb33070
+        applicationReference:
```

Comment:
`sourceReference` and `applicationReference` should be added to `required:` as well